WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/Preview/ProductDetailPreviewViewModel.swift

```diff
@@ -7,12 +7,29 @@ import protocol Storage.StorageManagerType
 /// View model for `ProductDetailPreviewView`
 ///
 final class ProductDetailPreviewViewModel: ObservableObject {
+    enum ProductDetailField: Equatable {
```

Comment:
Super nit: this could be renamed to `EditableField` for clarity. There's no need to repeat the `ProductDetail` prefix.

Reply:
Done in 9330760bd52d542cc68166a849e665e809dc632c